apps/src/code-studio/components/SoundPicker.jsx

```diff
@@ -74,23 +74,17 @@ export default class SoundPicker extends React.Component {
     };
 
     let modeSwitch;
-    let title = (
-      <p>
-        {this.props.assetChosen ? "Choose Sounds" : "Manage Sounds"}
-      </p>
-    );
+    let title = (<p>Choose Sounds</p>);
```

Comment:
Nit: Can we i18n 'Choose Sounds' while we're editing?

Reply:
Done!